### PR TITLE
Implement debug api to get forkchoice heads

### LINKED ISSUE #1824: Track all heads in fork choice
**Is your feature request related to a problem? Please describe.**
We'd like to implement the debug API endpoints, there is a 'get chain heads' method

**Describe the solution you'd like**
The fork choice should have a method to get the leaves (otherwise known as "heads")

**Additional context**
https://ethereum.github.io/eth2.0-APIs/#/Debug/getDebugChainHeads

### PR DESCRIPTION
**Pull request by twoeths**
resolves #1824 
+ Tested when syncing Pyrmont
Screen Shot 2020-12-08 at 13 22 22

### CODE REVIEW

File: packages/lodestar/test/unit/api/impl/debug/beacon/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    debugApi = new DebugBeaconApi(
+      {},
+      {
+        logger,
+        chain: chainStub,
+      }
+    );
+  });
+
+  it("should return head", async () => {
```
Comment (mpetrunic) [inline packages/lodestar/test/unit/api/impl/debug/beacon/index.test.ts:30]: add test case for null response

File: packages/lodestar/src/api/rest/controllers/debug/beacon/getHeads.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  handler: async function (req, resp) {
+    const heads = await this.api.debug.beacon.getHeads();
+    if (!heads) {
+      return resp.status(404).send();
+    }
+    const slotRootsType = new ListType({
+      elementType: this.config.types.SlotRoot,
+      limit: MAX_FORKCHOICE_HEADS,
+    });
```
Comment (mpetrunic) [inline packages/lodestar/src/api/rest/controllers/debug/beacon/getHeads.ts]: you could just iterate trough heads and serialize each of them without instantiating ListType

**Review by mpetrunic — COMMENTED** [review]
Nice, left just two small comments

**Review by mpetrunic — APPROVED** [review]

**Review by wemeetagain — APPROVED** [review]
:tada:
